frontend/benefit/applicant/src/components/applications/forms/application/step5/employeeView/EmployeeView.tsx

```diff
@@ -73,27 +73,35 @@ const EmployeeView: React.FC<EmployeeViewProps> = ({
                 }`
               )}
               {data.apprenticeshipProgram && (
-                <$ViewField>{`, ${data.paySubsidyPercent || ''} %`}</$ViewField>
+                <$ViewField isInline>{`, ${data.paySubsidyPercent || ''} % ${
```

Comment:
What is this comma `, ` for?

Reply:
its for second optional value. it can be Kyllä 30%, 40% or Kyllä 30%